packages/block-library/src/table/state.js

```diff
@@ -26,105 +28,186 @@ export function createTable( {
 }
 
 /**
- * Updates cell content in the table state.
+ * Returns the first row in the table.
  *
- * @param {Object} state               Current table state.
- * @param {string} options.section     Section of the cell to update.
- * @param {number} options.rowIndex    Row index of the cell to update.
- * @param {number} options.columnIndex Column index of the cell to update.
- * @param {Array}  options.content     Content to set for the cell.
+ * @param {Object} state Current table state.
  *
- * @return {Object} New table state.
+ * @return {Object} The first table row.
  */
-export function updateCellContent( state, {
-	section,
+export function getFirstRow( state ) {
+	if ( ! isEmptyTableSection( state.head ) ) {
+		return state.head[ 0 ];
+	}
+	if ( ! isEmptyTableSection( state.body ) ) {
+		return state.body[ 0 ];
+	}
+	if ( ! isEmptyTableSection( state.foot ) ) {
+		return state.foot[ 0 ];
+	}
+}
+
+/**
+ * Gets an attribute for a cell.
+ *
+ * @param {Object} state 			   	 Current table state.
+ * @param {string} options.sectionName	 Section of the cell to update.
+ * @param {number} options.rowIndex    	 Row index of the cell to update.
+ * @param {number} options.columnIndex 	 Column index of the cell to update.
+ * @param {number} options.attributeName The name of the attribute to get the value of.
+ *
+ * @return {*} The attribute value.
+ */
+export function getCellAttribute( state, {
+	sectionName,
 	rowIndex,
 	columnIndex,
-	content,
+	attributeName,
```

Comment:
I think this would read better if `attributeName` is a separate argument. E.g. `getCellAttribute( state, 'align', cell )`.

Reply:
Agreed it could be improved. I've changed it to `getCellAttribute( state, cell, 'align' )`.

Reply:
Great! It seems like a good idea to keep `cellLocation` as a separate entity. Could this object be documented somewhere to the type can be used in the docs of all these functions?